src/js/components/Layer/LayerContainer.js

```diff
@@ -163,7 +163,7 @@ const LayerContainer = forwardRef(
       content = (
         <FocusedContainer
           hidden={position === 'hidden'}
-          restrictScroll
+          restrictScroll={!layerTarget ? true : undefined}
```

Comment:
Can you add some comments here, just so we don't lose the "why"?

Reply:
I've added a comment for future reference.